package/com.unity.formats.usd/Runtime/Scripts/IO/SanitizedSamples.cs

```diff
@@ -0,0 +1,473 @@
+using System;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using pxr;
+using USD.NET;
+using USD.NET.Unity;
+using UnityEngine;
+
+[assembly: InternalsVisibleToAttribute("Unity.Formats.USD.Tests")]
+namespace Unity.Formats.USD
+{
+    public interface ISanitizable
```

Comment:
I love this idea - it awesomely simplifies import code and isolates the transformation into concise operators